src/test/java/webdrivertasks/page/SavedPasteBinPage.java

```diff
@@ -18,4 +19,18 @@ public boolean checkHeaderToTitleAccordance(String title) {
         return driver.getTitle().contains(title);
     }
 
+    public boolean checkBashPropertyIsActive(String bashPath) {
+        return new WebDriverWait(this.driver, 20)
+                .until(ExpectedConditions.presenceOfElementLocated(By.xpath(bashPath))).isDisplayed();
+    }
+
+    public boolean checkPastedToDisplayedTextAccordance(String askedCode) {
+        boolean codeCheckResult = true;
+        String pastedCode = new WebDriverWait(this.driver, 20)
```

Comment:
could you refactor existing tests with functionality from WebElementWaiting class?